src/user-defined-types/static.md

```diff
@@ -0,0 +1,38 @@
+---
+minutes: 5
+---
+
+# Static and Const
+
+Static and constant variables are two different ways to create globally-scoped
+values that cannot be moved or reallocated during the execution of the program.
```

Comment:
Can you remove this heading? This slide is about statics, and I think this little introductory header is unnecessary clutter.